Command/ListenCommand.php

```diff
@@ -105,12 +105,12 @@ private function start(ConnectionInterface $connection)
             $connection->getStreamingClient()->start();
         }
         catch (MemoryLimitException $e) {
-            $this->logger->critical($e->getMessage());
+            $this->logger->critical('ListenCommand-001 - '.$e->getMessage());
```

Comment:
Can you change this description?  Something like "Memory limit exception" or "ListenCommand: Memory limit Exception"